tools/backups2datalad.py

```diff
@@ -365,8 +369,6 @@ def get_annex_hash(filepath: Union[str, Path]) -> str:
                 # TODO/discussion: https://github.com/dandi/dandi-cli/issues/690
                 for f in ("dandiset_id", "version_id"):
                     adict.pop(f, None)
-                # and ensure that "modified" is a str
-                adict["modified"] = ensure_strtime(adict["modified"])
```

Comment:
100% no longer needed? ;-)

Reply:
`adict["modified"]` comes from `a.json_dict()`, which converts `datetime`s to `str`s.